src/test/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AFModelUnitTest.java

```diff
@@ -0,0 +1,246 @@
+                final AFParameters parameters = new AFParameters(f, pi, mu, beta);
+                for (final double n : Arrays.asList(1000, 2000, 3000)) {
+                    final int a = (int) (n * f / (f + (1 - f) * trueBias));
+                    final int r = (int) (n - a);
+                    final AllelicCount count = new AllelicCount(DUMMY, r, a);
+                    final double actualMean = AFModel.biasPosteriorMoment(parameters, count, AFIndicator.ALT_MINOR, 1);
+                    final double actualMeanSquare = AFModel.biasPosteriorMoment(parameters, count, AFIndicator.ALT_MINOR, 2);
+                    Assert.assertEquals(actualMean, trueBias, 1e-2);
+                    Assert.assertEquals(actualMeanSquare, trueBias*trueBias, 5e-2);
+                }
+            }
+        }
+    }
+
+    @Test
+    public void testMCMC() {
+        final int numSamples = 1000;
+        final int numBurnIn = 200;
+
+        final double averageHetsPerSegment = 20;
+        final int numSegments = 100;
+        final int averageDepth = 50;
+
+        final double meanBias = 1.1;
+        final double biasRate = 100;
+        final double pOutlier = 0.02;
+
+        // note: the following tolerances could actually be made much smaller if we used more segments and/or
+        // more hets -- most of the error is the sampling error of a finite simulated data set, not numerical error of MCMC
+        final double minorFractionTolerance = 0.03;
+        final double meanBiasTolerance = 0.03;
+        final double biasRateTolerance = 500;
+        final double pOutlierTolerance = 0.03;
+        AFSimulatedData truthData = new AFSimulatedData(averageHetsPerSegment, numSegments,
+                averageDepth, meanBias, biasRate, pOutlier);
+
+
+        final AFModel model = new AFModel(truthData.getData());
+        model.fitMCMC(numSamples, numBurnIn);
+
+
+        final List<Double> meanBiasSamples = model.getmeanBiasSamples();
+        Assert.assertEquals(meanBiasSamples.size(), numSamples);
+
+        final List<Double> biasRateSamples = model.getBiasRateSamples();
+        Assert.assertEquals(biasRateSamples.size(), numSamples);
+
+        final List<Double> pOutlierSamples = model.getOutlierProbabilitySamples();
+        Assert.assertEquals(pOutlierSamples.size(), numSamples);
+
+        final List<AFState.MinorFractions> minorFractionsSamples = model.getMinorFractionsSamples();
+        Assert.assertEquals(minorFractionsSamples.size(), numSamples);
+        for (final AFState.MinorFractions sample : minorFractionsSamples) {
+            Assert.assertEquals(sample.size(), numSegments);
+        }
+
+        final List<List<Double>> minorFractionsSamplesBySegment = model.getMinorFractionSamplesBySegment();
+
+        final double MCMCMeanBias = meanBiasSamples.stream().mapToDouble(x -> x).average().getAsDouble();
+        final double MCMCBiasRate = biasRateSamples.stream().mapToDouble(x -> x).average().getAsDouble();
```

Comment:
MCMCBiasRate not used, see below.  also, would be good to make these variables follow capitalization convention.